gigascience/gigadb-website
# Fix outstanding issues in view profile page

# Pull request for issue: #1546

This is a pull request for the following functionalities:

* Improve layout and fix outstanding a11y issues in http://gigadb.gigasciencejournal.com:9170/user/view_profile

## How to test?

* Navigate to http://gigadb.gigasciencejournal.com:9170/user/view_profile
* Run playwright tests (they all should pass)
* Run WAVE audit, there should be no "errors" reported
* Interact with tabs using the keyboard: arrow keys should navigate within tabs, tab key in and out
* Review appearance of readonly and edit states of user data

## How have functionalities been implemented?

* Updated tabs to fit tab pattern (https://www.w3.org/WAI/ARIA/apg/patterns/tabs/)
* refactored form by separating readonly data and form

## Any issues with implementation?

None

## Any changes to automated tests?

None

## Any changes to documentation?

None

## Any technical debt repayment?

None

## Any improvements to CI/CD pipeline?

None
